packages/web/src/components/withdraw-usdc-modal/components/EnterTransferDetails.tsx

```diff
@@ -80,21 +93,20 @@ export const EnterTransferDetails = () => {
   const handleAmountBlur: FocusEventHandler<HTMLInputElement> = useCallback(
     (e) => {
       setHumanizedValue(padDecimalValue(e.target.value))
-      setAmountTouched(true)
     },
-    [setHumanizedValue, setAmountTouched]
+    [setHumanizedValue]
   )
 
-  const [{ value: address }, { error: addressError }] = useField(ADDRESS)
+  const [
+    { value: address },
+    _ignoredAddressMeta,
+    { setTouched: setAddressTouched }
+  ] = useField(ADDRESS)
 
-  const handleClickHelpGuide = useCallback(() => {
-    track(
-      make({
-        eventName: Name.WITHDRAW_USDC_HELP_LINK_CLICKED,
-        currentBalance: analyticsBalance
-      })
-    )
-  }, [analyticsBalance])
+  const disableContinue =
+    methodValue === WithdrawMethod.COINFLOW
+      ? !!balance?.isZero()
+      : !!(!address || balance?.isZero())
```

Comment:
nit - maybe move the `!!` in front of `balance?.isZero()`? The triple `!` seemed weird